### DISCUSSION TITLE
dos2unix on which tools/push_firmware depends isn't compile when running "make tichksum-host"

### DISCUSSION THREAD

**Discussion by fraternl** [Freetz-NG/freetz-ng#1028] [Allgemein] [answered]
To create "tools/push_firmware" it is not needed to run a complete make to create an image.

```
 make tichksum-host
```
makes an almost-working flashing tool. The only thing that's missing is "**dos2unix**"

It would be nice if "**make tichksum-host**" would resolve that dependency.

**Comment by fda77** [comment, marked as answer]
09df838b9457c13b40cf48415d6f6cae47e6f53d
make "tichksum" still builds only tichksum

**Comment by fraternl** [comment]
No problem...
I can run:
```
make dos2unix-host
```
I must have made a typo when I tried to run that.

**Comment by fda77** [comment]
You could run `make tools` to build "all" (or unpack precompiled, if selected - in this case also optional)
